cpu/cc2538/startup-gcc.c

```diff
@@ -30,22 +30,23 @@
  *  OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 /**
- * \addtogroup cc2538dk
+ * \addtogroup cc2538
  * @{
  *
  * \file
- * Startup code for the cc2538dk platform, to be used when building with gcc
+ * Startup code for the cc2538 chip, to be used when building with gcc
  */
 #include "contiki.h"
 #include "reg.h"
 #include "flash-cca.h"
 #include "sys-ctrl.h"
-#include "uart.h"
 
 #include <stdint.h>
 /*---------------------------------------------------------------------------*/
 extern int main(void);
 /*---------------------------------------------------------------------------*/
+#define WEAK_ALIAS(x) __attribute__ ((weak, alias(#x)))
```

Comment:
Sorry, not many.

* `cpu/arm/stm32f103/startup-STM32F10x.c` uses `#define UNHANDLED_ALIAS`
* `cpu/stm32w108/hal/micro/cortexm3/stm32w108/crt-stm32w108.c` and `cpu/arm/stm32f103/startup-STM32F10x.c` also use aliases but without defining any macros